mvlearn/utils/plotting.py

```diff
@@ -0,0 +1,49 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+from .utils import check_Xs_y
+from .utils import check_Xs
+
+
+def plot_2views(
+    Xs,
```

Comment:
What are your thoughts on renaming this to something like "plot_2views_grid" or something that describes that it will do this grid of feature-wise plots? I could see us adding a subplot_nviews or something like that which actually plots each view separately (assuming they've been transformed to 2d), just to differentiate this from anything else in the future

Reply:
Or like plot_2views_cross

Reply:
Yes I'd prefer a new name. It reminds me of seaborn's pairplot but across views instead of within one. One of the following, plus/minus the word plot?

- crossviewplot? 
- twix_viewplot? 
- **crossplot**??? 
- pair2plot?
- pairedviews_plot?
- crossviews_plot?
- cross2views_plot?

I like crossplot personally but maybe because it's short and thus too vague.

Reply:
I like pairedviews_plot the most, then cross2views_plot or crossviews_plot

Reply:
duoviews_plot? i dont really like the 2 in the function name. crossviews_plot sounds good too

Reply:
I settled on crossviews_plot